src/main/java/pl/touk/parking/payment/DisabledDriverPaymentPolicy.java

```diff
@@ -0,0 +1,16 @@
+package pl.touk.parking.payment;
+
+import pl.touk.parking.domain.Money;
+
+public class DisabledDriverPaymentPolicy extends AbstractPaymentPolicy {
+
+    @Override
+    protected Money firstHourPayment() {
+        return Money.zero();
+    }
+
+    @Override
+    protected double thirdAndEachNextHourMultiplier() {
```

Comment:
double do takiego liczenia pieniędzy jest ok?

Reply:
No tak, ale dalej mnożysz pieniądze przez double ;) Może w tym przykładzie nic się nie zmieni, ale jakbyśmy robili więcej takich przeliczeń i potrzebowali większą precyzję, to pieniądze by się nam gubiły...